tests/test_benchmark_constraints.py

```diff
@@ -261,6 +263,85 @@ def test_benchmark_minimal_degree_of_autonomy_constraint(self, margs):
             degree_of_autonomy[use_case[0]] < degree_of_autonomy[use_case[1]]
         ), f"The  degree of autonomy of the scenario with the constraint is not higher than the one without, so the test does not make sense."
 
+    # this ensures that the test is only run if explicitly executed, ie not when the `pytest` command
+    # alone is called
+    @pytest.mark.skipif(
+        EXECUTE_TESTS_ON not in (TESTS_ON_MASTER),
+        reason="Benchmark test deactivated, set env variable "
+        "EXECUTE_TESTS_ON to 'master' to run this test",
+    )
+    @mock.patch("argparse.ArgumentParser.parse_args", return_value=argparse.Namespace())
+    def test_net_zero_energy_constraint(self, margs):
+        r"""
+        Notes
+        -----
+        With this benchmark test, the net zero energy (NZE) constraint is validated in a
+        single sector LES and a sector coupled LES.
+        Constraint_net_zero_energy_False contains a single sector LES without NZE constraint.
+        Constraint_net_zero_energy contains a single sector LES with NZE constraint.
+        Constraint_net_zero_energy_sector_coupled_False contains a sector-coupled LES without NZE constraint.
+        Constraint_net_zero_energy_sector_coupled contains a sector-coupled LES with NZE constraint.
+        The benchmark test passes if the degree of NZE of the defined energy systems
+        without constraint is lower than one and if the degree of NZE of the energy
+        systems with constraint equals one or is greater than one.
```

Comment:
You are also asserting the aggregated feedin into and consumption from grid, can add this to the list.

Reply:
done with fcc8e7f